content_scripts/vimium_frontend.coffee

```diff
@@ -21,8 +20,8 @@ isEnabledForUrl = true
 passKeys = null
 keyQueue = null
 # The user's operating system.
-currentCompletionKeys = null
-validFirstKeys = null
+currentCompletionKeys = ""
+validFirstKeys = ""
```

Comment:
Is there a good reason to do this? At the moment, initialising to `null` makes it easy to debug the case where something has gone wrong with getting these from the background (since we get an error message).

Also, `currentCompletionKeys` and `validFirstKeys` are both arrays when initialised, so this is misleading.